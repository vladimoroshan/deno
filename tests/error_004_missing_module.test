args: tests/error_004_missing_module.ts --reload
check_stderr: true
exit_code: 1
output: tests/error_004_missing_module.ts.out
