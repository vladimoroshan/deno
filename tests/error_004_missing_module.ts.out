Compiling [WILDCARD]tests/error_004_missing_module.ts
[WILDCARD]
Uncaught NotFound: Cannot resolve module "bad-module.ts" from "[WILDCARD]/tests/error_004_missing_module.ts"
    at DenoError ([WILDCARD]/js/errors.ts:[WILDCARD])
    at maybeError ([WILDCARD]/js/errors.ts:[WILDCARD])
    at maybeThrowError ([WILDCARD]/js/errors.ts:[WILDCARD])
    at sendSync ([WILDCARD]/js/dispatch.ts:[WILDCARD])
    at codeFetch ([WILDCARD]/js/os.ts:[WILDCARD])
    at resolveModule ([WILDCARD]/js/compiler.ts:[WILDCARD])
    at moduleNames.map.name ([WILDCARD]/js/compiler.ts:[WILDCARD])
    at resolveModuleNames ([WILDCARD]/js/compiler.ts:[WILDCARD])
    at compilerHost.resolveModuleNames ([WILDCARD]typescript.js:[WILDCARD])
    at resolveModuleNamesWorker ([WILDCARD]typescript.js:[WILDCARD])
